agent.go

```diff
@@ -179,7 +179,10 @@ func (a *Agent) crankParallel() error {
 			acc.Prefix = plugin.name + "_"
 			acc.Config = plugin.config
 
-			plugin.plugin.Gather(&acc)
+			err := plugin.plugin.Gather(&acc)
+			if err != nil {
+				log.Printf("Error in plugins: %s", err)
```

Comment:
Please add `plugin.name` to the debug message, something like 

`log.Printf("Error in plugin [%s]: %s", plugin.name, err)`